docs/tutorials/intro/inventory.ipynb

```diff
@@ -0,0 +1,1058 @@
+       "<span class=\"lineno\"> 4 </span>\n",
+       "<span class=\"lineno\"> 5 </span><span class=\"l l-Scalar l-Scalar-Plain\">global</span><span class=\"p p-Indicator\">:</span>\n",
+       "<span class=\"lineno\"> 6 </span>    <span class=\"l l-Scalar l-Scalar-Plain\">domain</span><span class=\"p p-Indicator\">:</span> <span class=\"l l-Scalar l-Scalar-Plain\">global.local</span>\n",
+       "<span class=\"lineno\"> 7 </span>    <span class=\"l l-Scalar l-Scalar-Plain\">asn</span><span class=\"p p-Indicator\">:</span> <span class=\"l l-Scalar l-Scalar-Plain\">1</span>\n",
+       "<span class=\"lineno\"> 8 </span>\n",
+       "<span class=\"lineno\"> 9 </span><span class=\"l l-Scalar l-Scalar-Plain\">eu</span><span class=\"p p-Indicator\">:</span>\n",
+       "<span class=\"lineno\">10 </span>    <span class=\"l l-Scalar l-Scalar-Plain\">asn</span><span class=\"p p-Indicator\">:</span> <span class=\"l l-Scalar l-Scalar-Plain\">65100</span>\n",
+       "<span class=\"lineno\">11 </span>\n",
+       "<span class=\"lineno\">12 </span><span class=\"l l-Scalar l-Scalar-Plain\">bma</span><span class=\"p p-Indicator\">:</span>\n",
+       "<span class=\"lineno\">13 </span>    <span class=\"l l-Scalar l-Scalar-Plain\">groups</span><span class=\"p p-Indicator\">:</span>\n",
+       "<span class=\"lineno\">14 </span>        <span class=\"p p-Indicator\">-</span> <span class=\"l l-Scalar l-Scalar-Plain\">eu</span>\n",
+       "<span class=\"lineno\">15 </span>        <span class=\"p p-Indicator\">-</span> <span class=\"l l-Scalar l-Scalar-Plain\">global</span>\n",
+       "<span class=\"lineno\">16 </span>\n",
+       "<span class=\"lineno\">17 </span><span class=\"l l-Scalar l-Scalar-Plain\">cmh</span><span class=\"p p-Indicator\">:</span>\n",
+       "<span class=\"lineno\">18 </span>    <span class=\"l l-Scalar l-Scalar-Plain\">asn</span><span class=\"p p-Indicator\">:</span> <span class=\"l l-Scalar l-Scalar-Plain\">65000</span>\n",
+       "<span class=\"lineno\">19 </span>    <span class=\"l l-Scalar l-Scalar-Plain\">vlans</span><span class=\"p p-Indicator\">:</span>\n",
+       "<span class=\"lineno\">20 </span>      <span class=\"l l-Scalar l-Scalar-Plain\">100</span><span class=\"p p-Indicator\">:</span> <span class=\"l l-Scalar l-Scalar-Plain\">frontend</span>\n",
+       "<span class=\"lineno\">21 </span>      <span class=\"l l-Scalar l-Scalar-Plain\">200</span><span class=\"p p-Indicator\">:</span> <span class=\"l l-Scalar l-Scalar-Plain\">backend</span>\n",
+       "</pre></div>\n",
+       "\n"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# groups file\n",
+    "%highlight_file ../../../examples/inventory/groups.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pretty similar to the hosts file.\n",
```

Comment:
Highlight the `defaults` key.

Reply:
So that's explained later in the "Inheritance Model" section. I am not sure if mentioning it here is too early and maybe overloading the reader with information is not yet relevant.